### DISCUSSION TITLE
Computer (Windows 10) no longer sleeping

### DISCUSSION THREAD

**Discussion by cjros** [kangyu-california/PersistentWindows#101] [Q&A] [answered]
Installed for Windows 10.
It seems that my computer no longer goes to sleep. 

I've always had the "High Performance" power plan and my computer would shut off the screen in one hour; computer would sleep in three hours.

After installing PW, it only turns off the screen but continues to stay on; I can hear windows notifications during the time it's supposed to sleep after three hours. Even coming back to the computer in the morning after being afk for the night, it just turns on the screen and the computer is just like it was before I left it; no sleeping, no login screen (like it usually did before)

I do have auto-restore enabled so is that what's keeping my computer awake the whole time?

**Comment by kangyu-california** [comment, marked as answer]
PW is purely event driven, so it is unlikely to cause PC to stay awake.
My laptop has no problem going to sleep, not sure the "Balanced" power setting is related or not.

When PC is going to sleep, PW would log the event as follows. PC wake up event is also logged.
![image](https://user-images.githubusercontent.com/59128756/112198203-195b4a80-8bca-11eb-8a8d-5f6c26b54bba.png)

I would definitely try "disable auto restore" first to see if it helps.
Then I would exit PW to confirm the issue is caused by PW. (you may capture windows to disk before exit, so that all window layout can be restored later on)

Run the latest 5.20 release might be helpful as well.
